src/locale/messages.fr.xlf

```diff
@@ -1,313 +1,365 @@
+        <target state="translated">Tous</target>
       </trans-unit>
       <trans-unit datatype="html" id="ae50a7291ac86efaefeefaaf53afff3b8b8c088b">
         <source>Digest</source>
-        <target state="new">Clé publique</target>
+        <target state="translated">Condensat</target>
       </trans-unit>
-      <trans-unit datatype="html" id="41a92d5e8a530001a958246408f411ccabfe3595">
+      <trans-unit datatype="html" id="aafb5b89839f9af24b0f7b61208d25361f11b293">
         <source>Digest required</source>
-        <target state="new">Drapeau (Digest) obligatoire</target>
+        <target state="translated">Le condensat obligatoire</target>
       </trans-unit>
       <trans-unit datatype="html" id="3530c857805fc74fe628dcc56456fa90ff9128d1">
         <source>Digest type</source>
-        <target state="new">Drapeau</target>
+        <target state="translated">Type de condensat</target>
       </trans-unit>
-      <trans-unit datatype="html" id="6decc246adf8fa54408bb99010015ff3fa27f059">
+      <trans-unit datatype="html" id="a8ccbfc31e1cfe3c97e9d017ca41a8f091d0a8d3">
         <source>Digest type required</source>
-        <target state="new">Drapeau obligatoire</target>
+        <target state="translated">Le type de condensat est obligatoire</target>
       </trans-unit>
-      <trans-unit datatype="html" id="2c098c2435a725b6528263e609a89a0ec3d188b2">
+      <trans-unit datatype="html" id="e6016cc16f7146d05fb4b3d8d4a9cf3aa894d8a6">
         <source>Disable IPv4</source>
-        <target state="new">Désactiver IPv4</target>
+        <target state="translated">Désactiver IPv4</target>
       </trans-unit>
-      <trans-unit datatype="html" id="da728aedfa61a52bd7b39213dc0ce50acb19ee26">
+      <trans-unit datatype="html" id="5fbd1b623677ff4abe8ff5a57603a17b2e5d2800">
         <source>Disable IPv6</source>
-        <target state="new">Désactiver IPv6</target>
+        <target state="translated">Désactiver IPv6</target>
       </trans-unit>
       <trans-unit datatype="html" id="a80323a72217e6e51ab903d5109e12c69ed70b79">
         <source>Domain name</source>
-        <target state="new">Nom de domaine</target>
+        <target state="translated">Nom de domaine</target>
       </trans-unit>
       <trans-unit datatype="html" id="d97fc0ce0c1d83a43448e9253f3b93f91b844b8c">
         <source>Domain name required</source>
-        <target state="new">Nom de domaine requis</target>
+        <target state="translated">Le nom de domaine obligatoire</target>
       </trans-unit>
-      <trans-unit datatype="html" id="5744111782246551528">
-        <source>Error during parent data fetching</source>
-        <target state="new">Erreur lors de la récupération des données parentes.</target>
+      <trans-unit datatype="html" id="4715502322841966813">
+        <source>Error during parent data fetching.</source>
+        <target state="translated">Erreur lors de la récupération des données parentes.</target>
```

Comment:
In order to be consistent with a sentence translated as “Les données de la zone parente ont été récupérées avec succès” below, I suggest this sentence be translated as “Erreur lors de la récupération des données de la zone parente.”

Reply:
I went for https://github.com/zonemaster/zonemaster-gui/pull/373#discussion_r1035900560